pan3d/dataset_builder.py

```diff
@@ -1,624 +1,291 @@
-import json
 import os
-import pandas
+import json
 import pyvista
-import typing
 import xarray
+
+from pan3d.utils import coordinate_auto_selection
 from pathlib import Path
 from pvxarray.vtk_source import PyVistaXarraySource
-from pyvista.trame.ui import plotter_ui
-
-from trame.decorators import TrameApp, change
-from trame.app import get_server
-from trame.widgets import html, client
-from trame.widgets import vuetify3 as vuetify
-import trame_server
-import trame_vuetify
-
-from pan3d.ui import AxisDrawer, MainDrawer, Toolbar, RenderOptions
-from pan3d.utils import (
-    initial_state,
-    has_gpu_rendering,
-    run_singleton_task,
-    coordinate_auto_selection,
-)
-
-BASE_DIR = Path(__file__).parent
-CSS_FILE = BASE_DIR / "ui" / "custom.css"
+from typing import Any, Dict, List, Optional, Union, Tuple
 
 
-@TrameApp()
 class DatasetBuilder:
-    """Manage data slicing, mesh creation, and rendering for a target N-D dataset."""
+    """Manage data structure, slicing, and mesh creation for a target N-D dataset."""
 
     def __init__(
         self,
-        server: typing.Union[trame_server.core.Server, str] = None,
         dataset_path: str = None,
-        state: dict = None,
+        server: Any = None,
         pangeo: bool = False,
     ) -> None:
         """Create an instance of the DatasetBuilder class.
 
         Parameters:
-            server: Trame server instance.
             dataset_path: A path or URL referencing a dataset readable by xarray.open_dataset()
-            state:  A dictionary of initial state values.
+            server: Trame server instance.
```

Comment:
`server: Trame server name or instance`

Reply:
Changed in [b87a945](https://github.com/Kitware/pan3d/pull/59/commits/b87a9450a989c0c672d41afe7d852b030323d219)